src/Lint.jl

```diff
@@ -25,9 +26,6 @@ const SIMILARITY_THRESHOLD = 10.0
 macro lintpragma(s)
 end
 
-# needed for BROADCAST
-include("compat.jl")
```

Comment:
We can delete the compat.jl file now, right?

Reply:
yup